ocfweb/docs/docs/staff/procedures/new-host.md

```diff
@@ -9,10 +9,51 @@ use hostnames of the form `hozer-{60..89}` and their corresponding IP addresses
 running `virsh undefine hozer-{num}` to remove the VM and `lvremove
 /dev/vg/hozer-{num}` to remove the logical volume.
 
-## Step 1. Pick a hostname and IP
 
-Add the hostname to the DNS. Contact the University hostmaster to request the
-reverse DNS record.
+## Step 0. Pick a hostname and IP
+
+If you are creating a brand-new host, you can find a list of IP addresses
+already in use in our [DNS repo on GitHub][github-ip-list]. Hostnames must be
+based on (un)natural disasters; a few previously used ones may be listed at the
+bottom of [our DNS template][github-dns-template].
+
+[github-ip-list]: https://github.com/ocf/dns/blob/master/etc/zones/db.ocf.in-addr.arpa
+[github-dns-template]: https://github.com/ocf/dns/blob/master/templates/db.ocf.tmpl
+
+
+## Step 1. (New hosts only) Add to LDAP, DNS, Puppet, Kerberos
+
+Only do these if a server with this hostname has never existed before (or if
+it's been long enough that some of these steps have never been done before).
+Unfortunately, these steps tend to change a lot as our infrastructure evolves.
+
+## Step 1.1. Add the LDAP entry
+
+On supernova, `kinit $USER/admin ldap-add-host <hostname> <ip>`. If creating a
+staff VM, also do `kinit $USER/admin ldapvi cn=<hostname>` and set the `type`
+attribute to `staffvm`.
+
+## Step 1.2. Add the DNS record
+
+Clone the [DNS repo][github-dns] from GitHub, run `build-zones`, and push a
+commit with the new records.
+
+Run `dns-reconcile` and request the needed DNS changes from the University
+hostmaster.
+
+[github-dns]: https://github.com/ocf/dns
+
+## Step 1.3. Add node config to Puppet
+
+Make a commit to the [Puppet repo][github-puppet] which adds a file
+`hieradata/nodes/<hostname.yaml>` for the new host. Follow the example of
+another server or staff VM's `host.yaml` file.
```

Comment:
only necessary if you need special config for the node, in general this is not necessary